1.7/administration/installing/custom/create-public-agent.md

```diff
@@ -29,44 +30,60 @@ You can determine the count of public agents in your cluster by running the foll
     $ sudo systemctl disable dcos-mesos-slave
     ```
 
-1.  Remove the old directory structures on the agent node.
+2.  Remove the old directory structures on the agent node.
 
     ```bash
-    $ sudo rm -rf /opt/mesosphere /var/lib/mesos
+    $ sudo rm -rf /etc/mesosphere /opt/mesosphere /var/lib/mesos
     ```
 
-1.  Restart the machine.
+3.  Restart the machine.
 
     ```bash
-    $sudo reboot
-    ```
+    $ sudo reboot
+    ```        
 
-### Install DC/OS
+### Install DC/OS and convert to a public agent node
+Copy the archived DC/OS installer file (`dcos-install.tar`) to the node that that is being converted to a public agent. This archive is created during the GUI or CLI [installation](/1.7/administration/installing/custom/gui/#backup) method.
 
-1.  Copy the `dcos-install.tar` file created in the GUI or CLI installation method to the machine that will be a public agent.
+1.  Copy the files to your agent node. For example, you can use Secure Copy (scp) to copy `dcos-install.tar` to your home directory:
 
-    ```bash
-    $ ssh $USER@$AGENT "sudo mkdir -p /opt/dcos_install_tmp"
-    $ scp ~/dcos-install.tar $ROOT_USER@$AGENT:~/dcos-install.tar
+    ```
+    $ scp ~/dcos-install.tar $username@$node-ip:~/dcos-install.tar
     ```
 
-1.  SSH to the machine:
+2.  SSH to the machine:
 
     ```bash
     $ ssh $USER@$AGENT
     ```
 
-1.  Install DC/OS as a public agent:
+1.  Create a directory for the installer files:
+
+     ```bash
+     $ sudo mkdir -p /opt/dcos_install_tmp
+     ```
+
+1.  Unpackage the `dcos-install.tar` file:
+
+    ```bash
+    $ sudo mv dcos-install.tar /opt/dcos_install_tmp
+    $ cd /opt/dcos_install_tmp
+    $ sudo tar xvf ~/opt/dcos_install_tmp/dcos-install.tar
```

Comment:
Might as well just do a target extract to get rid of the other two steps.

    tar xf dcos-install.tar -C /opt/dcos_install_tmp